catalog/spatial/registry/registry-api-impl/pom.xml

```diff
@@ -241,22 +241,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.75</minimum>
+                                            <minimum>0.50</minimum>
```

Comment:
I know brian is working unit tests for this class but since we added new functionality here we should probably have some new unit tests.